### ISSUE TITLE
Can't create new post

### ISSUE THREAD

**Issue by MiraNova23** [enchanted-sword/dashboard-unfucker#211]
**Describe the bug**
With new update to 5.6.2, can't create a new post. Tried to change settings, reload, new tabs, etc. No change.

**To Reproduce**
- Click "New Post" button in top right corner.
OR
- Click a "New [Type] Post" button from the carousel near the top.

**Screenshots**
Note the address says: https://www.tumblr.com/new/text 
But there is no pop up window for a new post at all.
![image](https://github.com/enchanted-sword/dashboard-unfucker/assets/154484784/cc02f137-dd33-4977-b62b-16b01d898475)

**Version (please complete the following information):**
 - FireFox 121.0 (64-bit)
 - Version 5.6.2

**Additional context**
Clicking the New Post button changes the address to https://www.tumblr.com/new
But clicking from the carousel changes the address to https://www.tumblr.com/new/text, or https://www.tumblr.com/new/quote, etc. Nothing produces a pop up window. Upon trying to reload the page, however, I do see a flash of one before the page reloads, with the same address, but still no visible pop up window.

**Comment by enchanted-sword** [comment]
Do images still open in lightbox view if you click on them?
![image](https://github.com/enchanted-sword/dashboard-unfucker/assets/81981111/44dd430c-8599-463b-9bb4-24b7da5631c2)

**Comment by MiraNova23** [comment]
I went to re-enable Tampermonkey, AND the script within Tampermonkey, but then realized that it didn't matter whether the Dashboard Unfucker script was enabled or not. That didn't seem right. I knew I had already checked any other possible problem extensions & had narrowed it down to this. But today I realized there was 1 other script in Tampermonkey that might be relevant: A login-*popup* blocker, that I thought was just for Pinterest. Turns out it includes many sites, like Tumblr, & has stopped development, so the latest Firefox must have broken *this one* actually.

The actual-problem script: https://greasyfork.org/en/scripts/395497-login-reminder-popup-remover-developement-ceased-not-up-to-date

So with that disabled, & Dashboard Unfucker re-enabled, it all works fine! Thank you for looking into this anyway!
